compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -413,66 +404,31 @@ def create_batch(self, task_group_id=None, create_websocket_queue=False) -> Batc
         Returns
         -------
         Batch instance
-            Status block containing "status" key.
         """
-        if not task_group_id:
-            task_group_id = self.session_task_group_id
-
-        return Batch(
-            task_group_id=task_group_id, create_websocket_queue=create_websocket_queue
-        )
+        return Batch(endpoint_id, task_group_id, create_websocket_queue)
 
     @requires_login
-    def batch_run(self, batch) -> t.List[str]:
-        """Initiate a batch of tasks to Globus Compute
+    def batch_run(self, batch: Batch) -> dict:
+        """
```

Comment:
Could be more specific in the return type, if you'd like.  The upstream API returns a structure of:
```
dict[str, str | dict[str, list[str]]]
```
And if Pydantic were in play here&nbsp;&mdash;&nbsp;I recognize it is not&nbsp;&mdash;&nbsp;we could further restrict that to:
```
dict[str, str | dict[uuid.UUID, list[uuid.UUID]]]
```